metadata/src/test/java/org/apache/kafka/controller/PartitionChangeBuilderTest.java

```diff
@@ -77,39 +77,75 @@ public void testChangeRecordIsNoOp() {
         );
     }
 
-    private final static PartitionRegistration FOO = new PartitionRegistration(
-        new int[] {2, 1, 3}, new int[] {2, 1, 3}, Replicas.NONE, Replicas.NONE,
-        1, LeaderRecoveryState.RECOVERED, 100, 200);
+    private static final PartitionRegistration FOO;
+
+    static {
+        try {
+            FOO = new PartitionRegistration.Builder().
+                setReplicas(new int[] {2, 1, 3}).
+                setIsr(new int[] {2, 1, 3}).
+                setRemovingReplicas(Replicas.NONE).
+                setAddingReplicas(Replicas.NONE).
+                setLeader(1).
+                setLeaderRecoveryState(LeaderRecoveryState.RECOVERED).
+                setLeaderEpoch(100).
+                setPartitionEpoch(200).
+                build();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Isn't `IllegalStateExection` a runtime exception?

Reply:
Yeah. I've changed the signature to not have `throws Exception` so I've removed the catches.